godot_ap/ui/console/base_console.gd

```diff
@@ -133,12 +133,17 @@ static func make_item(id: int, flags: int, data: DataCache) -> ConsoleLabel:
 	var color := AP.ComplexColor.as_rich(AP.get_item_class_color(flags))
 	return make_text(data.get_item_name(id), ttip, color)
 
-static func make_player(id: int) -> ConsoleLabel:
+static func make_player(id: int, own_id: int) -> ConsoleLabel:
```

Comment:
does `own_id` really need to be passed in? Can't you just read if off `Archipelago.conn.player_id` in this function?

Reply:
again, static function can't read non-static variables

Reply:
again, `Archipelago.`

Reply:
nvm, fixed